web/templates/setting/form.html.eex

```diff
@@ -0,0 +1,28 @@
+<%= form_for @conn, @action, [as: :settings, method: :put], fn f -> %>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for svc <- @system_variable_changesets do %>
+        <tr>
+          <td> <%= svc.data.key %> </td>
+
+          <%= inputs_for f, integer_to_atom(svc.data.id), fn ff -> %>
```

Comment:
Yes, so suppose you have a table with 10000 entries (which through the interface you can't create but you can do that accessing the database directly, which we can't guarantee it's not done). Each id is converted to an atom and placed in the atom's table. This may overflow the memory space allocated to the atom table which has a fixed amount and is not garbage collected.
I'm thinking about a solution, but can't find anything related in the form to replace the usage of atom for the fields. We can set an arbitrary number to limit the number of rows, but that's a monkey patch approach.